### PR TITLE
Adding capability support for LambdaTest

### PR DESCRIPTION
**Pull request by Shahnawaz-LambdaTest**
## Proposed changes
This PR adds the capability support found in [LambdaTest Automation Capability Generator](https://www.lambdatest.com/capabilities-generator/) with the addition of `LambdaTestCapabilities` interface.

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: packages/wdio-types/src/Capabilities.ts [hunk 1]
```diff
@@ -154,6 +154,8 @@ export interface VendorExtensions extends EdgeCapabilities, AppiumW3CCapabilitie
     'sauce:visual'?: SauceLabsVisualCapabilities
     // Experitest Access Keys
     'experitest:accessKey'?: string
+    //LambdaTest w3c specific
+    'LT:Options'?: LambdaTestCapabilities
```
Comment (christian-bromann) [inline packages/wdio-types/src/Capabilities.ts:158]: Is lowercase/uppercase important here? I would recommend to keep caps lowercased so wanted to make sure this isn't a platform requirement

File: packages/wdio-types/src/Capabilities.ts [hunk 2]
```diff
@@ -1071,6 +1073,73 @@ export interface SauceLabsCapabilities {
...
+    tunnel?: boolean
+    
+    /**
+     * Capture browser console logs at various steps in the test.
+     */
+    console?: "warn" | "error" | "warn" | "info" | "true" 
+
+    network?: boolean
+
+}
```
Comment (christian-bromann) [inline packages/wdio-types/src/Capabilities.ts:1141]:
```suggestion
export interface LambdaTestCapabilities{
    username?: string
    accessKey?: string
    platformName?: string
    browserName?: string
    browserVersion? : string
    /**
     * Set the resolution of the VM.
     */
    resolution?: string
    selenium_version?: string
    headless?: boolean
    seCdp?: boolean
    /**
     * Specify a name for a logical group of builds.
     */
    project?: string
    /**
     * Specify a name for a logical group of tests.
     */
    build?: string | number
    /**
     * Use this capability to add a custom tag to the builds.
     * These tags can be used to filter the builds on the Automate dashboard.
     */
    buildTags?: Array<string>
    smartUiProject?: string
    /**
     * Use this capability to add names to the tests.
     */
    name?: string
    /**
     * Use this capability to add a custom tag to the tests.
     * These tags can be used to filter the tests on the Automate dashboard.
     */
    tags?: Array<string>
    visual?: boolean
    video?: boolean
    /**
     * Test locally hosted websites on LambdaTest.
     * To enable access to the local machine you need to setup the
     * LambdaTest Tunnel (https://www.lambdatest.com/support/docs/testing-locally-hosted-pages).
     */
    tunnel?: boolean
    /**
     * Capture browser console logs at various steps in the test.
     */
    console?: "warn" | "error" | "warn" | "info" | "true" 
    network?: boolean
}
```

**Review by christian-bromann — COMMENTED** [review]
One comment

**Comment by christian-bromann** [comment]
Also, mind signing the CLA? If you like this to be backported to v7 faster, please raise a PR to the `v7` branch as well.

File: packages/wdio-types/src/Capabilities.ts (hunk 1 above)
Comment (Shahnawaz-LambdaTest) [reply packages/wdio-types/src/Capabilities.ts:158]: This must be in uppercase as this is a LambdaTest platform requirement.

Comment (christian-bromann) [reply packages/wdio-types/src/Capabilities.ts:158]: Ouch, ok

**Comment by christian-bromann** [comment]
@Shahnawaz-LambdaTest once you can sign the CLA I can merge and release this.

**Comment by Shahnawaz-LambdaTest** [comment]
> @Shahnawaz-LambdaTest once you can sign the CLA I can merge and release this.
> @christian-bromann I have signed the CLA, Please merge the PR and release.

**Pull request merged**